integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KafkaClusterConfig.java

```diff
@@ -138,6 +136,23 @@ public Stream<ConfigHolder> getBrokerConfigs(Supplier<KafkaEndpoints> endPointCo
                 server.put(String.format("listener.name.%s.plain.sasl.jaas.config", "EXTERNAL".toLowerCase()), plainModuleConfig);
             }
 
+            if (securityProtocol != null && securityProtocol.contains("SSL")) {
+                KeytoolCertificateGenerator keytoolCertificateGenerator = new KeytoolCertificateGenerator();
+                try {
+                    keytoolCertificateGenerator.generateSelfSignedCertificateEntry("test@redhat.com", clientEndpoint.getConnect().getHost(), "KI", "RedHat", null, null,
+                            "US");
+                }
+                catch (GeneralSecurityException | IOException e) {
+                    throw new RuntimeException(e);
+                }
+                server.put("ssl.client.auth", "required");
+                server.put("ssl.truststore.location", KeytoolCertificateGenerator.getCertLocation());
+                server.put("ssl.truststore.password", KeytoolCertificateGenerator.getPassword());
+                server.put("ssl.keystore.location", KeytoolCertificateGenerator.getCertLocation());
+                server.put("ssl.keystore.password", KeytoolCertificateGenerator.getPassword());
+                server.put("ssl.key.password", KeytoolCertificateGenerator.getPassword());
```

Comment:
Can we pull this out as a private method as it looks to be repeated a couple of times.

Reply:
It is used in a similar way, but not for the same purpose. First, it is set first for `server`, that is a `Properties` variable and then for `kafkaConfig` that is a `HashMap`. Then, they all are not always set for clients, because it depends on the `securityProtocol` set, but for servers are always all set.

Reply:
Urgh that's an annoying duplication 😞